CHANGELOG.md

```diff
@@ -5,6 +5,34 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## Unreleased
+
+### Removed
+* ESP-IDF v4.3 support, including mostly conditional compilations. (#420)
+### Deprecated
+**ESP-IDF v4.4** Please start upgrading to ESP-IDF v5.
+### Changed (Breaking)
+* GPIO20 is now included on esp32. Notice that not every esp32 has a physical GPIO20. (#370)
+* ledc: now it can also be used with high_speed channels on esp32. (#424)
+* dependency: updated embassy-sync to 0.6. (#437)
+* adc: legacy adc driver is now behind a feature flag `adc-oneshot-legacy` starting with >= ESP-IDF v5.0. For justification, look at https://github.com/espressif/esp-idf/issues/13938. (#433)
+* spi: cs_pre/post_trans_delay() config option included. (#266)
+### Added
+* rmt: `Symbol` now derives `Clone` and `Copy` (#386)
+* reset: restart() function (#383)
+* pcnt: Can now be used with esp32c6. (#407)
+* can: Frame flags enum (#411)
+* task: added MallocCap enum - Flag to indicate the capabilities of a Memory Region. (#419)
+* i2s: new I2sDriver implementation allowing for splited bidirectional i2s. (#435)
+### Fixed
+* esp32h2 builds: added missing hys_ctrl_mode field (#387)
+* rmt: FixedLengthSignal was broken, which led to failures in the neopixel/smartled examples, among other things. (#402)
+* e-hal 0.2: implements DelayUs and DelayMs for u8 to be consistent with the other implementations (#414)
+* i2c: wrong timeout calculations. (#415)
+* task: Compatibility with ESP-iDF v5-3 (pre-release) - adding stack_alloc_caps to `ThreadSpawnConfiguration`. (#419)
+* i2s: Compatibility with ESP-IDF v5.3 (pre-release) - small internal adjustments to i2s (#419)
```

Comment:
i moved that to Fixed - since its all related to the pre-release version, though technically its adding a field for that version we up until now didn't support ;D So one could argue that its a breaking change ( only for the new v5.3 versions) but its not breaking for any older version so i think that's why it make sense to put it into fixed.